napari/resources/build_icons.py

```diff
@@ -132,6 +133,46 @@ def build_resources_qrc(
     return qrc_path, theme_dir
 
 
+def _find_rcc_or_raise() -> str:
+    """Locate the Qt rcc binary to generate resource files
+
+    1. we always want to use pyrcc5 if it's available, regardless of API
+    2. it will sometimes, (if not always) be named pyrcc5.bat on windows...
+       but shutil.which() will find that too
+    3. We also want to prefer binaries higher up on the path, and we add
+       sys.executable to the front of the path (and \\Scripts on windows)
+    4. after pyrcc5 we try pyside2-rcc
+
+    see https://github.com/napari/napari/issues/1221
+    and https://github.com/napari/napari/issues/1254
+
+    Returns
+    -------
+    path : str
+        Path to the located rcc binary, or None if not found
+
+    Raises
+    ------
+    FileNotFoundError
+        If no executable can be found.
+    """
+    python_dir = os.path.dirname(sys.executable)
```

Comment:
@tlambert03 re #1336, should this have been `os.path.abspath(os.path.join(os.path.dirname(sys.executable), '..'))`

Reply:
No, it shouldn't, never mind. :thinking: